src/main/resources/application-development.properties

```diff
@@ -1,6 +1,6 @@
 logging.level.sql=DEBUG
 logging.level.org.hibernate.type.descriptor.sql.BasicBinder=TRACE
-spring.datasource.url=jdbc:h2:file:./target/db-development
+spring.datasource.url=jdbc:h2:file:./target/db-development;AUTO_SERVER=TRUE
```

Comment:
Can you add something to the PR description about this?

And consider possibly making it a separate PR; that will both benefit the team because you'll get more points!  And it's the right thing to do, industry practice wise.

Reply:
Merged, so you can leave this because when we rebase it won't show up anymore.